src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -126,4 +132,11 @@ class EthService(blockchain: Blockchain, appStateStorage: AppStateStorage) {
       highestBlock = appStateStorage.getEstimatedHighestBlock()))
   }
 
+  def sendRawTransaction(req: SendRawTransactionRequest): Future[SendRawTransactionResponse] = {
+    import SignedTransactions._
+    val signedTransaction = rlp.decode[SignedTransaction](req.data.toArray[Byte])
```

Comment:
I think are right, my mistake while reading geth code :+1:
I just tested this with geth and it does return various different error messages when the input doesn't represent an encoded tx.